### PR TITLE
program-website-pp initial commit.

### PR DESCRIPTION
**Pull request by a3l5o7n9**
- Program-website design implementation
- Program-website and Profile-page navigation implementation.

### CODE REVIEW

File: projects/angular/program-website/src/app/app.component.html [hunk 1]
```diff
@@ -1,13 +1 @@
...
-  <div>
-    <span>Title: </span>
-    <span>{{ book.getTitle() }}</span>
-  </div>
-  <div>
-    <span>Amount of pages: </span>
-    <span>{{ book.getPages() }}</span>
-  </div>
-</div>
+<router-outlet></router-outlet>
```
Comment (webdevelopland) [inline projects/angular/program-website/src/app/app.component.html]: New line

File: projects/angular/program-website/src/app/core/services/firebase.service.ts [hunk 2]
```diff
@@ -3,47 +3,45 @@ import { AngularFirestore, AngularFirestoreCollection } from '@angular/fire/fire
...
 export class FirebaseService {
-  private protobin: AngularFirestoreCollection<FirebaseElement>;
+  private dataSource: AngularFirestoreCollection<FirebaseElement>;
 
   constructor(
     private db: AngularFirestore,
     private encodingService: EncodingService,
   ) {
-    this.protobin = this.db.collection('protobin');
+      this.dataSource = this.db.collection('reviewer');
```
Comment (webdevelopland) [inline projects/angular/program-website/src/app/core/services/firebase.service.ts]: Why space?

File: projects/angular/program-website/src/app/core/services/firebase.service.ts [hunk 3]
```diff
@@ -3,47 +3,45 @@ import { AngularFirestore, AngularFirestoreCollection } from '@angular/fire/fire
...
       .snapshotChanges()
       .pipe(
         map(action => {
-          const firebaseElement = action.payload.data() as FirebaseElement;
-          if (firebaseElement === undefined) {
-            // Element not found
-            return;
-          }
-          return this.convertFirebaseElementToBook(firebaseElement);
+        const firebaseElement: FirebaseElement = action.payload.data() as FirebaseElement;
```
Comment (webdevelopland) [inline projects/angular/program-website/src/app/core/services/firebase.service.ts]: Please try to format code here.
`Ctrl` + `Shitft` + I

File: projects/angular/program-website/src/app/routes/home/home.component.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+            </a>
+          </div>
+          <div class="contact-phone">
+            <p>
+              012-3456789
+            </p>
+          </div>
+        </div>
+      </div>
+      <img src="../assets/images/image-4.svg" class="image-4">
```
Comment (webdevelopland) [inline projects/angular/program-website/src/app/routes/home/home.component.html]: Why do you use `../`?

File: projects/angular/program-website/src/app/routes/home/home.component.scss [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+  border-radius: 33px;
+  border: solid 2px #de4e34;
+  background-color: #ffffff;
+  >.read-more-link {
+    font-size: 20px;
+    font-weight: 800;
+    color: #de4e34;
+    text-decoration: none;
+  }
+}
```
Comment (webdevelopland) [inline projects/angular/program-website/src/app/routes/home/home.component.scss]: New line

File: projects/angular/program-website/src/assets/cv-pages/cv-template.css [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+  PADDING: 0px 15px 0px 0px;
+  MARGIN: 0px;
+  VERTICAL-ALIGN: top;
+  FONT-SIZE: 8pt;
+}
+
+.space {
+  DISPLAY: BLOCK;
+  HEIGHT: 5px;
+}
```
Comment (webdevelopland) [inline projects/angular/program-website/src/assets/cv-pages/cv-template.css]: new line

File: projects/angular/program-website/src/app/core/services/firebase.service.ts (hunk 2 above)
Comment (a3l5o7n9) [reply projects/angular/program-website/src/app/core/services/firebase.service.ts]: Indentation.

Comment (a3l5o7n9) [reply projects/angular/program-website/src/app/core/services/firebase.service.ts]: `Ctrl` + `Shift` + I removed the space.

File: projects/angular/program-website/src/app/routes/home/home.component.html (hunk 4 above)
Comment (a3l5o7n9) [reply projects/angular/program-website/src/app/routes/home/home.component.html]: Is it possible to use `@/`?
I don't know how.

Comment (a3l5o7n9) [reply projects/angular/program-website/src/app/routes/home/home.component.html]: Nevermind. Just had to remove `../` from the `src` string.

**Pull request merged**